CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fixed a bug in topological ordering module. (#442)
+- Fixed a bug in coercion of empty string to boolean value. (#453)
```

Comment:
Version 0.1.2 is already released. Please move new changes to the [Unreleased] sectuin